beamer/themes/theme/ep.jhu.edu/beamerthemeep.jhu.edu.dtx

```diff
@@ -210,6 +213,274 @@
+      \includegraphics[
+          height=\paperheight,
+          keepaspectratio,
+          width=\paperwidth,
+      ]{GilmanHall}
+    };
+    \shade[
+        left color=PMS 284C,
+        opacity=0.85,
+        right color=Pantone 7683 C,
+    ] (current page.north west) rectangle (current page.south east)+(-0.25\paperwidth,0);
+    \node at (current page.center) {
+      \includegraphics[
+          width=0.6\paperwidth,
+      ]{{whiting.logo.small.vertical.white}.eps}
+    };
+  \end{tikzpicture}%
+  \usebeamertemplate{copyright}%
+}
+%    \end{macrocode}
+%
+% \subsubsection{Frame Title}
+% Add rule underneath the frame title.
+%    \begin{macrocode}
+\defbeamertemplate*{frametitle}{ep.jhu.edu}{%
+  \vskip2ex%
+  \insertframetitle%
+  \par%
+  {\color{PMS 284C}\rule{2em}{0.5ex}}%
+}
+%    \end{macrocode}
+%
+% \subsubsection{Footline}
+% Customize the footline with the logo and frame number.
+%    \begin{macrocode}
+\defbeamertemplate*{footline}{ep.jhu.edu}{%
+  \ifbeamer@plainframe%
+  \else%
+    \leavevmode%
+    \hbox{%
+      \begin{beamercolorbox}[
+          dp=\beamerthemeepjhuedu@footline@depth,
+          left,
+          leftskip=\Gm@lmargin,  % https://tex.stackexchange.com/a/198224
+          wd=0.33\textwidth,
+      ]{logo in head/foot}%
+        \insertlogo%
+      \end{beamercolorbox}%
+      \begin{beamercolorbox}[
+          wd=0.34\textwidth,
+      ]{}%
+      \end{beamercolorbox}%
+      \begin{beamercolorbox}[
+          dp=\beamerthemeepjhuedu@footline@depth,
+          right,
+          rightskip=\Gm@rmargin,  % https://tex.stackexchange.com/a/198224
+          wd=0.33\textwidth,
+      ]{page number in head/foot}%
+        \vbox to 4ex{% https://tex.stackexchange.com/a/49617
+          \vfil%
+          \insertframenumber%
+          \vfil%
+        }%
+      \end{beamercolorbox}%
+    }%
+    \vskip0pt%
+  \fi%
+}
+%    \end{macrocode}
+%
+% \subsubsection{Sidebar Right}
+% Omit the logo (as compared to the \textsf{default} outer theme).\footnote{\url{https://github.com/josephwright/beamer/blob/d6af506/base/themes/outer/beamerouterthemedefault.sty\#L121}}.
+%    \begin{macrocode}
+\defbeamertemplate*{sidebar right}{ep.jhu.edu}{%
+  \vfill%
+  \llap{\usebeamertemplate***{navigation symbols}\hskip0.1cm}%
+  \vskip2pt%
+}
+%    \end{macrocode}
+%
+% \subsubsection{Title Page}
+% The title page template features Gilman Hall in the background with the Whiting School of Engineering logo prominently displayed in the top left corner.
+%    \begin{macrocode}
+\defbeamertemplate*{title page}{ep.jhu.edu}[1][]{%
+%    \end{macrocode}
+% Suppress the headline, footline, and sidebars on the current frame by manually setting the corresponding key.
+% This approach may not be generalizable (e.g., if the theme has a headline, then setting this key inside this template is likely too late to have the desired effect), but works well for the \textsf{default} outer theme that does not define a headline, footline, or left sidebar.
```

Comment:
Stack Exhange questions:

* [How to remove footer for specific type of slides/frames?](https://tex.stackexchange.com/q/427257)
* [Beamer titlepage without the headline/footline template](https://tex.stackexchange.com/q/600218)